recipe/meta.yaml

```diff
@@ -113,20 +107,18 @@ outputs:
         ###################################################
         # run-like
         - alembic >=1.13.0
-        - astropy >=6.0.0,<6.1.5
-        - astroquery >=0.4.6,<0.4.8
+        - astropy-base >=7.0.1, <8
+        - astroplan >=0.10.1
+        - astroquery >=0.4.6
```

Comment:
Astroquery is a bit of a risk since they don't promise any compatibility at all between releases. Regardless I think the min pin should be >=0.4.9 now because that should work fine and they fixed some of the compatibility issues that broke 0.4.8. The safe thing to do is to pin ==0.4.9 so let's do that unless @ktlim decides we should let it float up until we get breakage.